### DISCUSSION TITLE
这个怎么解

### DISCUSSION THREAD

**Discussion by jian-li-5205** [share121/inter-knot#2701] [其他]
![1723968040669](https://github.com/user-attachments/assets/d70f0ce2-6470-4659-a8ad-852d78d44194)

**Comment by Defihhhhh** [comment]
这个委托我接了

**Reply by homteam** [reply]
666

**Reply by JobNumber-9527** [reply]
byd只接委托不解答是吧

**Comment by AAAzhaoxiangji** [comment]
这个委托我解了（doge

**Comment by homteam** [comment]
总所周知e^iπ+1=0，所以1=0-e^iπ，再通过勾股定理a^2+b^2=c^2得出没有e^iπ这个选项，所以1=0,1+1=0+0=0.🤯
（此答案来自作业帮bushi）

**Reply by cyberrobber** [reply]
wc，怎么算出来的，详细讲讲

**Comment by 662494** [comment]
你试试换个方向看看，1+1，王，对吧，所以1+1=王

**Comment by terrestrial19** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/4d5296bde00b3a911735a7599b9dc72e_3884466188049147.png" alt="彦卿 我来了" width="75" >这个委托是我的

**Comment by lllliufeng** [comment]
1 + 1 = 10（确信
<div align="right">——世界上有10种人，一种懂二进制，一种不懂</div>
